Refresh.GameServer/Database/GameDatabaseContext.Users.cs

```diff
@@ -380,4 +387,16 @@ public void IncrementUserFilesizeQuota(GameUser user, int amount)
             user.FilesizeQuotaUsage += amount;
         });
     }
+    
+    public void SetPrivacySettings(GameUser user, MetadataEndpoints.PrivacySettings settings) 
```

Comment:
Ouch. Yeah, I would extract PrivacySettings to its own class. `SerializedPrivacySettings` somewhere in `Types.UserData`.

Generally speaking the database should never depend on endpoints, and ideally never depend on Bunkum or any services. Reason being I hope to someday move the database schema to its own project in the solution.